proof/crefine/AARCH64/Wellformed_C.thy

```diff
@@ -562,6 +562,50 @@ lemma ucast_irq_array_guard[unfolded irq_array_size_val, simplified]:
   done
 
 
+text \<open>cacheLineBits interface\<close>
+
+lemmas cacheLineBits_val =
+  cacheLineBits_def[unfolded Kernel_Config.CONFIG_L1_CACHE_LINE_SIZE_BITS_def]
+
+lemma cacheLineBits_leq_pageBits:
+  "cacheLineBits \<le> pageBits"
+  using cacheLineBits_sanity
+  by (simp add: pageBits_def)
+
+lemma pageBits_leq_pptrBaseOffset_alignment:
+  "pageBits \<le> pptrBaseOffset_alignment"
```

Comment:
I'm a bit lost, is pptrBaseOffset_alignment related to cache line size?

Reply:
yes, it's larger :-) 

The relationship is that `addrFromPPtr` alignment is preserved by anything smaller or equal to `pptrBaseOffset_alignment`, which is what we need for the `.. && mask cacheLineBits = ...` lemma